docs/next/guides/building-and-testing/modules.md

```diff
@@ -481,352 +275,822 @@ new Handsontable(container, {});
+
+// register the `password` editor
 registerEditor(PasswordEditor);
 ```
 
-Now, you can use `PasswordEditor`, the full example looks like this:
+::: tip
+#### Registering all editors at once
+
+If you import all of Handsonable's editor modules, register them all with the `registerAllEditors()` method:
+
+```js
+// import the `registerAllEditors()` method
+import { registerAllEditors } from 'handsontable/registry';
+
+// register all editors
+registerAllEditors();
+```
+:::
+
+#### Step 5: Use your editor
+
+Now, you can start using your editor, by setting the [`editor`](@/api/options.md#editor) [configuration option](@/guides/getting-started/setting-options.md):
 
 ```js
 import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
 import { registerEditor, PasswordEditor } from 'handsontable/editors';
 
 registerEditor(PasswordEditor);
 
-// use the editor as needed
+// an example
 new Handsontable(container, {
   columns: [
-    {
+    { 
+      // use the `password` editor
       editor: 'password',
     },
   ]
-  // rest of the settings
 });
 ```
 
-And that is all! You can use the password editor!
+### Using renderer modules
+
+Each renderer module contains a different type of [cell renderer](@/guides/cell-functions/cell-renderer.md):
+
+| Module                 | Cell renderer alias |
+| ---------------------- | ------------------- |
+| `autocompleteRenderer` | `autocomplete`      |
+| `baseRenderer`         | `base`              |
+| `checkboxRenderer`     | `checkbox`          |
+| `htmlRenderer`         | `html`              |
+| `numericRenderer`      | `numeric`           |
+| `passwordRenderer`     | `password`          |
+| `textRenderer`         | `text`              |
 
-#### Importing all editors at once
+To start using a renderer module, follow the steps below.
 
-You can quickly import and register all editors at once:
+#### Step 1: Import the required modules
+
+First, make sure to [import the required modules](#importing-required-modules):
 
 ```js
-// import the `registerAllEditors()` function
-import { registerAllEditors } from 'handsontable/registry';
+// import the `handsontable/base` module
+import Handsontable from 'handsontable/base';
 
-// register all editors
-registerAllEditors();
+// import Handsontable's CSS
+import 'handsontable/dist/handsontable.full.css';
 ```
 
-## Importing renderers
+#### Step 2: Import the renderers' registering method
 
-To use a specific renderer you need to import it alongside the registering method. For instance, let's try adding the autocomplete renderer.
+Import the `registerRenderer()` method that lets you register your chosen renderer:
 
-Start with importing the base, `autocompleteRenderer` and the `registerRenderer` method.
+```js
+import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
+
+// import the `registerRenderer()` method
+import { registerRenderer } from 'handsontable/renderers';
```

Comment:
I don't see how a comment like the one on line 558 is useful. Obvious things should not be stated in code comments.

Reply:
Removing, as commented [here](https://github.com/handsontable/handsontable/pull/9059#discussion_r789751566)